tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -124,7 +124,7 @@
     "## Single-Qubit Gates <a name=\"single_gates\"/>\n",
```

Comment:
<p>Can you please add the generate matrix form for U gate here as well? You can find it on the UGate API documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Should be addressed now.